File: DefaultCodeInputOutput/10.js
const longestCommonPrefix = (strs) => {
  //Write your code here;
};

const main = () => {
  if (process.argv.length < 3) {
      console.error(JSON.stringify({ error: "No input provided" }));
      process.exit(1);
  }

  const inputStr = process.argv[2];
  let strs;
  try {
      const data = JSON.parse(inputStr);
      if (!data || typeof data !== 'object' || !Array.isArray(data.strs)) {
          throw new Error();
      }
      strs = data.strs;
      if (!strs.every(s => typeof s === 'string')) {
          throw new Error();
      }
  } catch (e) {
      console.error(JSON.stringify({ error: "Invalid input" }));
      process.exit(1);
  }

  const result = longestCommonPrefix(strs);
  console.log(JSON.stringify({ longest_common_prefix: result }));
};

main();

File: DefaultCodeInputOutput/10.py
import json
import sys

def longest_common_prefix(strs):
    #Write your code here

if __name__ == "__main__":
    if len(sys.argv) < 2:
        print(json.dumps({"error": "No input provided"}))
        sys.exit(1)

    input_str = sys.argv[1]
    try:
        strs = json.loads(input_str)
        if not isinstance(strs, list) or not all(isinstance(s, str) for s in strs):
            raise ValueError
    except ValueError:
        print(json.dumps({"error": "Invalid input"}))
        sys.exit(1)

    result = longest_common_prefix(strs)
    print(json.dumps({"result": result}))
